force-app/main/default/classes/Einstein_PredictionService.cls

```diff
@@ -835,8 +835,18 @@ public class Einstein_PredictionService {
     }
     
     public static String getAccessToken() {
+        // Try to retrieve email setting
         Einstein_Settings__c settings = Einstein_Settings__c.getOrgDefaults();
-        ContentVersion base64Content = [SELECT Title, VersionData FROM ContentVersion where Title='einstein_platform' LIMIT 1];
+        if (settings.Einstein_EMail__c == null)
+            throw new Einstein_ConfigurationException('Missing Einstein email setting. Check your Custom Settings.');
+        ContentVersion base64Content;
```

Comment:
@pozil The formatting is off here.

Reply:
@pozil Please add also test methods so that code coverage isn't affected negatively.